components/onboarding/ScheduleAndAvailability.tsx

```diff
@@ -4,13 +4,21 @@ import RHFCheckbox from '../RHFInputs/RHFCheckbox';
 import RHFDatePicker from '../RHFInputs/RHFDatePicker';
 import { Button } from '../ui/button';
 
+import { useFormContext, useWatch } from 'react-hook-form';
+
 // ----------------------------------------------------------------
 
 const ScheduleAndAvailability: React.FC = () => {
+  const { getValues } = useFormContext();
+  useWatch({ name: 'startDate' });
```

Comment:
The reason why I user useWatch API is because I don't get up to date value without it. 
examples.

I set the end date to 20 July and the for the start date i can choose up to 20 July. Then if I set end date to 27th July limit for the start date stays at 20 July. With useWatch hook it works as expected.. Without it it looks like getValues gets fired only on the first time. I found this solution on web.